metrics/network/network-nginx-ab-benchmark.sh

```diff
@@ -69,10 +69,16 @@ function nginx_ab_networking {
 	sleep "$sleep_secs"
 	ab -n ${requests} -c ${concurrency} http://${url}/ > "$total_requests"
 
-	result=$(cat "$total_requests" | grep "Requests per second" | cut -d ':' -f2 | sed 's/ //g' | cut -d '[' -f1)
+	cp -p $total_requests result.test
+
+	local result=$(grep "^Requests per second" $total_requests)
+	[[ $result =~ :[[:blank:]]*([[:digit:]]+(\.[[:digit:]]*)?) ]] && rps=${BASH_REMATCH[1]}
```

Comment:
I avoided leaning toothpicks :) I considered pasting sample data in a comment above, but realized I'd probably have to sanitize that since it's actual benchmarking results.